plugin/audit_log/audit_log.c

```diff
@@ -798,11 +839,14 @@ void audit_log_notify(MYSQL_THD thd __attribute__((unused)),
           strncmp(event_general->general_command, "Quit", 4) == 0)
         break;
 
+      if (local->skip_logging)
```

Comment:
It depends on what we want to do actually. If we want to skip connect/disconnect/change user events, then yes. If we don't want then we can only move it before `strncmp(event_general->general_command.str, "Quit", 4)`